config/config.ini.php

```diff
@@ -364,6 +364,18 @@
 ; For instance if your URL is 'http://example.com/directory/subdirectory', set urlpath to 'directory/subdirectory'.
 urlpath =
 
+; username_min_length (Integer)
+; The minimum character length a username can be.
+; This length is check once the username has been trimmed and multiple spaces have been collapsed into one.
+; Minimum for this config value is 1. Maximum is 150 as the resulting profile URL shouldn't be longer than 255 chars.
```

Comment:
Not "shouldn't" but "mustn't".

Reply:
Oh, now we are shooting for RFC 2119 compliance? https://www.ietf.org/rfc/rfc2119.txt

Reply:
It's more like making it clear for everyone ;-)